backend/src/main/java/org/eclipse/tractusx/puris/backend/common/edc/logic/util/EdcRequestBodyBuilder.java

```diff
@@ -140,9 +141,42 @@ public JsonNode buildBpnRestrictedPolicy(Partner partner) {
         return body;
     }
 
+    /**
+     * Creates a request body in order to register a policy that
+     * allows only participants of the framework agreement.
+     *
+     * @return the request body
+     */
+    public JsonNode buildFrameworkAgreementPolicy() {
+        var body = MAPPER.createObjectNode();
+        var context = MAPPER.createObjectNode();
+        context.put("odrl", ODRL_NAMESPACE);
+        body.set("@context", context);
+        body.put("@id", FRAMEWORK_POLICY_ID);
+        var policy = MAPPER.createObjectNode();
+        body.set("policy", policy);
+        policy.put("@type", "Policy");
+        var permissionsArray = MAPPER.createArrayNode();
+        policy.set("odrl:permission", permissionsArray);
+        var permissionsObject = MAPPER.createObjectNode();
+        permissionsArray.add(permissionsObject);
+        permissionsObject.put("odrl:action", "USE");
```

Comment:
Please use `use` instead of `USE`

Reply:
I tried that. But unfortunately the TractusX EDC version (0.5.3), on which we are currently working, won't work properly, when "use" instead of "USE" is inserted there. But we should keep that in mind, in case we move on to a newer EDC version.

Reply:
In such a case, this issue should be addressed at the Tractus-X EDC. But it shouldn't block this PR.